packages/client/src/util/vkt.ts

```diff
@@ -19,7 +19,7 @@ export async function generateVKTStateRoot(genesisState: GenesisState, common: C
   for (const addressStr of Object.keys(genesisState)) {
     const addrState = genesisState[addressStr]
     let nonce, balance, code
-    let storage: StoragePair[] = []
+    let storage: StoragePair[] | undefined = []
```

Comment:
This is initialized to an empty array so why can it also be typed as `undefined`?

Reply:
Nvm I see now, from the alloc it can be written as undefined. However can we then not just use 
```suggestion
    let storage: StoragePair[]
```

Not sure might be invalid syntax (add a `?` maybe)